demo/Gemfile.lock

```diff
@@ -213,8 +213,5 @@ DEPENDENCIES
   web-console (>= 3.3.0)
   webpacker (~> 5.0)
 
-RUBY VERSION
```

Comment:
This will cause problems as this will flag a diff every time. If you use appraisals it's actually easier than this way. In fact you don't even need to use appraisals if you wanted manually to upload the gemfiles (Just appraisals regenerates them for you)